climada_petals/hazard/tc_tracks_forecast.py

```diff
@@ -83,6 +87,18 @@
 MISSING_LONG = ec.CODES_MISSING_LONG
 """Missing double and integers in ecCodes """
 
+CXML2CSV_XSL = Path(__file__).parent / "data/cxml_ecmwf_transformation.xsl"
+"""Xsl file for transforming CXML to CSV format."""
```

Comment:
@emanuel-schmid I have now created a data folder in hazard and moved this transformation file therein. I am not sure whether this is the best place or if it should go somewhere else. I have tried to move it in my climada/data folder but then I get an error from jenkins that it does not find the file (I think I should have also changed something in the CONFIG file but I am not sure what).